src/lib/components/LeafletMap/README.md

```diff
@@ -0,0 +1,940 @@
+# 🌍 LeafletMap Component
+The new leafletMap component is a component for layered map data, like tile-data and webgl images. 
+
+## 📋 Table of contents
+- [Usage](#usage)
+    <br>
+- [Props](#props)
+    - [Map specifics](#props-map-specifics)
+    - [Map tools](#props-map-tools)
+    - [Map syncronization](#props-map-syncronization)
+
+    <br>
+- [Layers](#layers)
+    - [Structure](#structure)
+    - [Layer types](#layer-types)
+        - [Image layers](#image-layers)
+        - [Tile layers](#tile-layers)
+        - [Shapes and drawings](#shapes-and-drawings)
+    <br>    
+    - [Updating layers](#updating-layers)
+        - [With action](#updating-layers-with-action)
+        - [With replace](#updating-layers-with-replace)
+    <br>
+    - [Color scales](#colorscales)
+    - [Shaders](#shaders)
+    <br>
```

Comment:
Does this markdown render correctly? Remove `<br>`s?
![image](https://user-images.githubusercontent.com/31612826/89002005-3f12a700-d2fc-11ea-8357-0b460089750e.png)